internship2018-1/src/index.html

```diff
@@ -1,23 +1,128 @@
     </head>
     <body>
-        <h1>This is main header of my page. Это сайт не кодировке, עִבְרִית</h1>
-        <h3>This is our sub*3 header </h3>
-        <h6>This is our sub*6 header </h6>
-        <p>
-            Hello World!
-        </p>
+        <!-- Navigation start -->
+        <div class="container">
+                <nav class="navbar navbar-expand-lg navbar-light bg-light">
+                    <a class="navbar-brand" href="./index.html">
+                        <img src="/assets/image.png" width="30" height="30" alt="">
+                        Softheme
+                    </a>
+                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                        <div class="navbar-nav">
+                        <a class="nav-item nav-link active" href="./index.html">Home</a>
+                        <a class="nav-item nav-link" href="./lesson3.html">Lesson 3</a>
+                        <a class="nav-item nav-link" href="./products.html">Products</a>
+                        </div>
+                    </div>
+                </nav>           
+            </div>
+            <!-- Navigation end -->
+
+        <div class="container index">
+
+                <h1>Bootstrap responsive design</h1>
 
-        <p>
-            This is another paragraph with tag p
+                <h2>Grid system</h2>
+    
+                <h3>1. Auto layout columns</h3>
+                <div class="row">
+                    <div class="col">col1</div>
+                    <div class="col">col2</div>
+                    <div class="col">col3</div>
+                    <div class="col">col4</div>
+                    <div class="col">col5</div>
+                    <div class="col">col6</div>
+                    <div class="col">col7</div>
+                    <div class="col">col8</div>
+                    <div class="col">col9</div>
+                    <div class="col">col10</div>
+                </div>
+    
+                <h3>2. Setting one column width</h3>
+                <div class="row">
+                    <div class="col">col 1</div>
+                    <div class="col-6">col 2</div>
+                    <div class="col">col 3</div>
+                </div>
+    
+                <h3>3. Variable width content</h3>
+                <div class="row">
+                    <div class="col-12 col-md-2">col 1</div>
+                    <div class="col-12 col-md-7">col 2</div>
+                    <div class="col-12 col-md-3">col 3</div>
+                </div>
+    
+                <h3>4. Repeatable content</h3>
+                <div class="row">
+                    <div class="col-12 col-sm-4">col 1</div>
```

Comment:
col-12 means one product for one line by default
col-sm-4 overrides default settings with 3 products for one line (4 for each * 3 = 12 cols)
Use this approach for product page